src/server/pegasus_server_impl.cpp

```diff
@@ -2274,6 +2289,33 @@ void pegasus_server_impl::update_replica_rocksdb_statistics()
     dinfo_replica("_pfc_rdb_bf_point_negatives: {}", bf_point_negatives);
 }
 
+void pegasus_server_impl::update_replica_rocksdb_amp_statistics()
+{
+    // Update _pfc_rdb_write_amplification
+    std::map<std::string, std::string> props;
+    if (_db->GetMapProperty(_data_cf, "rocksdb.cfstats", &props)) {
+        auto write_amplification_iter = props.find("compaction.Sum.WriteAmp");
+        auto write_amplification = write_amplification_iter == props.end()
+                                       ? 1
+                                       : std::stod(write_amplification_iter->second);
+        _pfc_rdb_write_amplification->set(write_amplification);
+        dinfo_replica("_pfc_rdb_write_amplification: {}", write_amplification);
+    }
+
+    // Update _pfc_rdb_read_amplification
+    if (FLAGS_read_amp_bytes_per_bit > 0) {
+        auto estimate_useful_bytes =
+            _statistics->getTickerCount(rocksdb::READ_AMP_ESTIMATE_USEFUL_BYTES);
```

Comment:
https://github.com/facebook/rocksdb/blob/v6.6.4/monitoring/statistics.cc
I notice that there are other statistics similar to what we like, such as `GET_HIT_L0`, `GET_HIT_L1`, `GET_HIT_L2_AND_UP`, what it the difference between them?

Reply:
Oh, I haven't noticed, it seem hitRate under different level

Reply:
Yes~ I think they are actually what we need, more accurate than read amplification rate

Reply:
https://github.com/facebook/rocksdb/blob/v6.6.4/include/rocksdb/statistics.h#L101-L106 it hit count, we need calc `hitRate` via them